packages/workspace/src/command-line/output.ts

```diff
@@ -0,0 +1,183 @@
+import chalk from 'chalk';
+
+export interface CLIErrorMessageConfig {
+  title: string;
+  bodyLines?: string[];
+  slug?: string;
+}
+
+export interface CLIWarnMessageConfig {
+  title: string;
+  bodyLines?: string[];
+  slug?: string;
+}
+
+export interface CLILogMessageConfig {
+  title: string;
+  bodyLines?: string[];
+}
+
+export interface CLINoteMessageConfig {
+  title: string;
+}
+
+export interface CLISuccessMessageConfig {
+  title: string;
+}
+
+class CLIOutput {
+  private readonly NX_PREFIX = `${chalk.cyan(
+    '>'
+  )} ${chalk.reset.inverse.bold.cyan(' NX ')}`;
+  /**
+   * Longer dash character which forms more of a continuous line when place side to side
+   * with itself, unlike the standard dash character
+   */
+  private readonly VERTICAL_SEPARATOR =
+    '———————————————————————————————————————————————';
+
+  /**
+   * Expose some color and other utility functions so that other parts of the codebase that need
+   * more fine-grained control of message bodies are still using a centralized
+   * implementation.
+   */
+  colors = {
+    gray: chalk.gray
+  };
+  bold = chalk.bold;
+  underline = chalk.underline;
+
+  private writeToStdOut(str: string) {
+    process.stdout.write(str);
```

Comment:
Currently, we use `console.log` and `console.error`. Node sends `console.log` into stdout, and `console.error` into stderr. 

I think the difference only matters if someone pipes the result of running the command to another command, which I don't think anyone does. I don't think we use them consistently not do I think the angular cli uses them consistently either. 

This PR changes this. We no longer use stderr.

Reply:
It's probably for the best cause at least we make it consistent

Reply:
We discussed this and decided to press ahead for now, with the CI=true solution for no formatting applied